sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/JmsIOTest.java

```diff
@@ -214,7 +214,6 @@ public void testReadMessages() throws Exception {
     producer.send(message);
     producer.send(message);
     producer.send(message);
-    producer.send(message);
```

Comment:
out of curiosity, why remove one message here?

Reply:
becaues that makes it 5, otherwise the tests won't pass :)

Reply:
haha, I see L229 now, which means this test never worked before